docpipeline/generate.py

```diff
@@ -78,7 +79,7 @@ def clone_templates(dir):
 def process_blob(blob, credentials, devsite_template):
     log.info(f"Processing {blob.name}...")
 
-    tmp_path = pathlib.Path("tmp")
+    tmp_path = pathlib.Path(tempfile.TemporaryDirectory().name)
```

Comment:
Do we need a `with` clause to make this get automatically cleaned up?

Reply:
No need, Python will automatically delete the temporary directory (`TemporaryDirectory` being one of the API from `tempfile` that does that) without being in a `with` clause